jatszohaz/templates/static_pages/about_us.html

```diff
@@ -16,7 +16,10 @@ <h1>{% trans "About us" %}</h1>
       js.src = 'https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.12&appId=214800381898510&autoLogAppEvents=1';
       fjs.parentNode.insertBefore(js, fjs);
     }(document, 'script', 'facebook-jssdk'));</script>
-    <div class="text-center">
-        <br/><div class="fb-page" data-href="https://www.facebook.com/SchJatszohaz/" data-small-header="false" data-adapt-container-width="true" data-hide-cover="false" data-show-facepile="true"><blockquote cite="https://www.facebook.com/SchJatszohaz/" class="fb-xfbml-parse-ignore"><a href="https://www.facebook.com/SchJatszohaz/">Schönherz Játszóház</a></blockquote></div>
-    </div>
+            <div class="text-center">
+                <br/><div style="float:left; width: 50%;" class="fb-page" data-href="https://www.facebook.com/SchJatszohaz/" data-small-header="false" data-adapt-container-width="true" data-hide-cover="false" data-show-facepile="true"><blockquote cite="https://www.facebook.com/SchJatszohaz/" class="fb-xfbml-parse-ignore"><a href="https://www.facebook.com/SchJatszohaz/">Schönherz Játszóház</a></blockquote></div>
+                <div style="display: inline-block; width: 30%; height: 130px; padding: 18px 0 0 0;">
+                    <a href="https://discord.gg/WS3VtB7ynP"><img src="../../static/img/Discord_Logo.png" width="100%" /></a>
+                </div>
+            </div>

```

Comment:
Az invite link meddig érvényes? Nem fog pár héten belül lejárni?

Tabolás javítás:
```suggestion

    <div class="text-center">
    <br/><div style="float:left; width: 50%;" class="fb-page" data-href="https://www.facebook.com/SchJatszohaz/" data-small-header="false" data-adapt-container-width="true" data-hide-cover="false" data-show-facepile="true"><blockquote cite="https://www.facebook.com/SchJatszohaz/" class="fb-xfbml-parse-ignore"><a href="https://www.facebook.com/SchJatszohaz/">Schönherz Játszóház</a></blockquote></div>
        <div style="display: inline-block; width: 30%; height: 130px; padding: 18px 0 0 0;">
            <a href="https://discord.gg/WS3VtB7ynP"><img src="../../static/img/Discord_Logo.png" width="100%" /></a>
        </div>
    </div>
```

Reply:
Elvileg olyan invite linket generáltattam ami nem jár le.